webapp/utils/constants.jsx

```diff
@@ -865,6 +865,8 @@ export const Constants = {
     DEFAULT_MAX_NOTIFICATIONS_PER_CHANNEL: 1000,
     MAX_TEAMNAME_LENGTH: 15,
     MAX_TEAMDESCRIPTION_LENGTH: 50,
+    MIN_CHANNELNAME_LENGTH: 2,
+    MAX_CHANNELNAME_LENGTH: 15,
```

Comment:
Minor nit: the maximum channel name length is actually [22](https://docs.mattermost.com/help/getting-started/organizing.html?highlight=character#the-importance-of-channel-names), not 15.

Also, is the `MAX_CHANNELNAME_LENGTH` constant used in the code?